[2022-08-06 19:37:27,990] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 19:37:28,010] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 19:37:28,010] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:28,010] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:28,010] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:28,047] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-17 00:00:00+00:00
[2022-08-06 19:37:28,054] {standard_task_runner.py:52} INFO - Started process 1432 to run task
[2022-08-06 19:37:28,058] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-17T00:00:00+00:00', '--job-id', '426', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpxs75wrj3', '--error-file', '/tmp/tmpr26qceft']
[2022-08-06 19:37:28,058] {standard_task_runner.py:80} INFO - Job 426: Subtask second_task
[2022-08-06 19:37:28,138] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:28,221] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-17T00:00:00+00:00
[2022-08-06 19:37:28,222] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:28,223] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:28,237] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:28,238] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:28,239] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:28,271] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211217T000000, start_date=20220806T193727, end_date=20220806T193728
[2022-08-06 19:37:28,313] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:28,341] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:23,575] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 21:51:23,593] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 21:51:23,593] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:23,593] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:23,593] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:23,615] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-17 00:00:00+00:00
[2022-08-06 21:51:23,624] {standard_task_runner.py:52} INFO - Started process 1625 to run task
[2022-08-06 21:51:23,628] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-17T00:00:00+00:00', '--job-id', '429', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpjv5u1wd3', '--error-file', '/tmp/tmpxauu1y4l']
[2022-08-06 21:51:23,629] {standard_task_runner.py:80} INFO - Job 429: Subtask second_task
[2022-08-06 21:51:23,742] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:23,869] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-17T00:00:00+00:00
[2022-08-06 21:51:23,870] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:23,871] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:23,893] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:23,894] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:23,894] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:23,972] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211217T000000, start_date=20220806T215123, end_date=20220806T215123
[2022-08-06 21:51:24,002] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:24,061] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:47,569] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 21:56:47,589] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 21:56:47,589] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:47,589] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:47,589] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:47,609] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-17 00:00:00+00:00
[2022-08-06 21:56:47,617] {standard_task_runner.py:52} INFO - Started process 1422 to run task
[2022-08-06 21:56:47,621] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-17T00:00:00+00:00', '--job-id', '427', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpw063c5vw', '--error-file', '/tmp/tmp2f1pgeeo']
[2022-08-06 21:56:47,622] {standard_task_runner.py:80} INFO - Job 427: Subtask second_task
[2022-08-06 21:56:47,699] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:47,779] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-17T00:00:00+00:00
[2022-08-06 21:56:47,780] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:47,781] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:47,793] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:47,795] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:47,795] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:47,830] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211217T000000, start_date=20220806T215647, end_date=20220806T215647
[2022-08-06 21:56:47,873] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:47,910] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:48,413] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 22:11:48,426] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 22:11:48,427] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:48,427] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:48,427] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:48,442] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-17 00:00:00+00:00
[2022-08-06 22:11:48,472] {standard_task_runner.py:52} INFO - Started process 1413 to run task
[2022-08-06 22:11:48,475] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-17T00:00:00+00:00', '--job-id', '425', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpot7payd1', '--error-file', '/tmp/tmpgqe9htny']
[2022-08-06 22:11:48,475] {standard_task_runner.py:80} INFO - Job 425: Subtask second_task
[2022-08-06 22:11:48,537] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:48,608] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-17T00:00:00+00:00
[2022-08-06 22:11:48,609] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:48,609] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:48,621] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:48,623] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:48,623] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:48,647] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211217T000000, start_date=20220806T221148, end_date=20220806T221148
[2022-08-06 22:11:48,686] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:48,712] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:11,201] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 22:23:11,214] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [queued]>
[2022-08-06 22:23:11,214] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:11,214] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:11,214] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:11,229] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-17 00:00:00+00:00
[2022-08-06 22:23:11,236] {standard_task_runner.py:52} INFO - Started process 1482 to run task
[2022-08-06 22:23:11,239] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-17T00:00:00+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp5zr4i6_5', '--error-file', '/tmp/tmpyxk5sxw9']
[2022-08-06 22:23:11,239] {standard_task_runner.py:80} INFO - Job 432: Subtask second_task
[2022-08-06 22:23:11,330] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-17T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:11,467] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-17T00:00:00+00:00
[2022-08-06 22:23:11,468] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:11,469] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:11,488] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:11,490] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:11,490] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:11,535] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211217T000000, start_date=20220806T222311, end_date=20220806T222311
[2022-08-06 22:23:11,576] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:11,665] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
